benefits/templates/admin/login.html

```diff
@@ -0,0 +1,28 @@
+{% extends "admin/login.html" %}
+{% load i18n static %}
+
+{% block dark-mode-vars %}
+{% endblock dark-mode-vars %}
+
+{% block title %}
+    Log in | Cal-ITP Benefits Administrator
+{% endblock title %}
+
+{% block extrastyle %}
```

Comment:
Maybe add a comment here to say why we're not bringing in jQuery here / why we're using our own `extrastyle` block here instead of extending from `agency-base.html`?

Reply:
Good idea! Added here: https://github.com/cal-itp/benefits/pull/2371/commits/27bbf27b7c66b926f7a09c964ed8794b3c7628c6